__tests__/app.test.js

```diff
@@ -118,4 +118,64 @@ describe('3. Get api/articles/:article_id', () => {
                 })
         })
     })
+})
+
+describe('4. GET /api/articles/:article_id/comments', () => {
+    describe('a. status 200 & data', () => {
+        it('returns 200 status code & an array back to the user where the data starts with the latest comment', () => {
+            return request(app)
+            .get('/api/articles/3/comments')
+            .expect(200)
+            .then((res) => {
+                const articles = res.body;
+                expect(articles).toBeInstanceOf(Array);
+                expect(articles.length).toBe(2);
+                articles.forEach(article => {
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            article_id : 3,
+                            author : expect.any(String),
+                            body : expect.any(String),
+                            created_at : expect.any(String),
+                            votes : expect.any(Number)
+                        })
+                    )
+                })
+                expect(articles).toBeSorted({ descending: true })

```

Comment:
can you specify what the articles are being sorted by? Have a look at .toBeSortedBy instead